lesson03/main.js

```diff
@@ -0,0 +1,20 @@
+const ul = document.getElementById("js-target");
+
+const li = [];
+const a = [];
+const img = [];
+
+for (let i=0; i < 2; i++){
+  li[i] = document.createElement("li");
+  a[i] = document.createElement("a");
+  img[i] = document.createElement("img");
+
+  a[i].href = `a${i}.html`;
+  a[i].textContent = `a${i}`;
+
```

Comment:
IMO
If we create the contents of the array first and loop through it, we can do the following here.
```suggestion
  const li = document.createElement("li");
  const a = document.createElement("a");
  const img = document.createElement("img");

  a.href = `a${i}.html`;
  a.textContent = `a${i}`;
  a.src = imgSrcs[i];
```

Reply:
Thank you so much mai-san :)
Actually It's simple :)

Reply:
I fixed my code :)
9d6d14096c5c71b89a30101de0a90edb02284144